auth_user_case_insensitive/models/res_users.py

```diff
@@ -32,3 +32,24 @@ def write(self, vals):
         if vals.get('login'):
             vals['login'] = vals['login'].lower()
         return super(ResUsers, self).write(vals)
+
+    @api.model
+    def search(self, args, offset=0, limit=None, order=None, count=False):
+        """
```

Comment:
Format this according to PEP rules for doc comments (using full sentences, If possible one line, etc.).

Reply:
now compliant with https://www.python.org/dev/peps/pep-0257/